src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -231,6 +246,10 @@ private static String getBuildKey(@NonNull Run<?, ?> build, String branch,
             key = build.getParent().getFullName(); // use the job full name as the key for the status
         }
 
+        if (includeJenkinsURL) {
+            key = rootUrl + "/" + key;
```

Comment:
There might be double `/` depending on the user input - but I figured it doesn't makes any difference, the string is just a seed for a hash - as long as it's consistent, we're good.